docs/commands.md

```diff
@@ -3214,10 +3214,14 @@ hal config provider kubernetes account add ACCOUNT [parameters]
 When no context is configured for an account the 'current-context' in your kubeconfig is assumed.
  * `--deployment`: If supplied, use this Halyard deployment. This will _not_ create a new deployment.
  * `--docker-registries`: (*Default*: `[]`) A list of the Spinnaker docker registry account names this Spinnaker account can use as image sources. These docker registry accounts must be registered in your halconfig before you can add them here.
+ * `--kinds`: (*Default*: `[]`) (V2 Only) A list of kinds this Spinnaker account can deploy to and will cache.
```

Comment:
"A list of Kubernetes resource kinds this Spinnaker will be restricted to deploying and caching"? Even following the link it may not be immediately obvious what a "kind" is to a Kubernetes novice.